.github/workflows/test-all-custom-flags.yml

```diff
@@ -235,7 +235,7 @@ jobs:
 
       - name: Communicate BIT failure on originating issue
         uses: actions/github-script@f28e40c7f34bde8b3046d885e986cb6290c5673b # v7.1.0
-        if: failure() && steps.set-send-pr-feedback.outputs.send_pr_feedback == 'true'
+        if: (failure() && steps.get-build-artifact.conclusion == 'success')) && steps.set-send-pr-feedback.outputs.send_pr_feedback == 'true'
```

Comment:
Do not report back failures if fetching the build artifact failed (`clients` already has merge prevention for build failures, so we don't need to test/report these cases).